src/EnergyPlus/UnitarySystem.cc

```diff
@@ -8533,7 +8535,7 @@ namespace UnitarySystems {
                                                       CompressorONFlag);
                         PartLoadRatio = CoolPLR;
                     } else if (DataGlobals::DoCoilDirectSolutions && CoolingLoad &&
-                               this->m_CoolingCoilSubType_Num == DataHVACGlobals::CoilDX_SubcoolReheat && this->m_NumOfSpeedCooling == 1) {
+                               this->m_CoolingCoilSubType_Num == DataHVACGlobals::CoilDX_Cooling && this->m_NumOfSpeedCooling == 1) {
```

Comment:
I had to make this change manually after pulling in develop.  I think I have it wrong though.  This should also be checking the new `SubcoolReheatFlag`, right?

Reply:
@Myoldmopar Yes. I am going to work on it on my local computer and let you know what happens.